docs/prms/allpairs.rst

```diff
@@ -8,3 +8,17 @@ with respect to a cartesian product.
 
 All Pairs does not take any arguments. Its source code is at https://github.com/Reed-CompBio/all-pairs-shortest-paths
 licensed under MIT.
+
+Dataset Usage
+-------------
+
+All Pairs Shortest Paths uses ``sources``, ``targets``, and edge weights.
+All Pairs Shortest Paths also can incorporate graph directionality: it accepts directed, undirected, and mixed graphs.
```

Comment:
I think just means this just means it accepts mixed graphs. Otherwise be clear that the graph is fully directed or fully undirected.